ovirt/resource_ovirt_vm.go

```diff
@@ -376,6 +382,71 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 	}
 	vmBuilder.Cpu(cpu)
 
+	if storage_domain, _ := blockDevice.([]interface{})[0].(map[string]interface{})["storage_domain"]; storage_domain != "" && templateIDOK {
+
+		// Get the reference to the service that manages the storage domains
+		sdsService := conn.SystemService().StorageDomainsService()
+
+		// Find the storage domain we want to be used for virtual machine disks
+		sdsResp, err := sdsService.List().Search("name=" + storage_domain.(string)).Send()
+		if err != nil {
+			return fmt.Errorf("Failed to search storage domains, reason: %v", err)
+		}
+
+		tds, err := getTemplateDiskAttachments(templateID.(string), meta)
+		if err != nil {
+			return fmt.Errorf("Failed to get Template disks attachments: %v", err)
+		}
+
+		if sdSlice, ok := sdsResp.StorageDomains(); ok {
```

Comment:
rename sdSlice to storgeDomains, as this is not a slice. It's confusing to read the next line `sdSlice.Slice()` instead of `storageDomains.Slice()`

Reply:
Variable renamed - https://github.com/oVirt/terraform-provider-ovirt/pull/216/commits/dde1f319d7d24865125051d15c3a49d7307d4eac